### ISSUE TITLE
Verify that Travis supports configured Perl versions

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-backend#740]
At the time of writing the [.travis.yml] (develop branch) includes
```
  include:
  # Cover MySQL and PostgreSQL with the latest supported Perl version
  - perl: "5.32"
    env: TARGET=MySQL ZONEMASTER_RECORD=0 ZONEMASTER_BACKEND_CONFIG_FILE=./share/travis_mysql_backend_config.ini
    services: mysql
```
but Travis does not support Perl 5.32. It says
```
Downloading archive: https://storage.googleapis.com/travis-ci-language-archives/perl/binaries/ubuntu/16.04/x86_64/perl-5.32.tar.bz2
curl: (22) The requested URL returned error: 404 Not Found
$ sudo tar xjf perl-5.32.tar.bz2 --directory /
tar (child): perl-5.32.tar.bz2: Cannot open: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now
rm: cannot remove 'perl-5.32.tar.bz2': No such file or directory
$ travis_apt_get_update
$ sudo systemctl start mysql
$ git clone --depth=50 https://github.com/zonemaster/zonemaster-backend.git zonemaster/zonemaster-backend

Setting environment variables from .travis.yml
$ export TARGET=MySQL
$ export ZONEMASTER_RECORD=0
$ export ZONEMASTER_BACKEND_CONFIG_FILE=./share/travis_mysql_backend_config.ini
0.08s$ perlbrew use 5.32

ERROR: The installation "5.32" is unknown.

$ perl --version

This is perl 5, version 22, subversion 1 (v5.22.1) built for x86_64-linux-gnu-thread-multi
(with 73 registered patches, see perl -V for more detail)
```

To discover this discrepancy you have to inspect the Travis output, because Travis does not report this as an error.

Make sure that this reported as an error by Travis.

[.travis.yml]: https://github.com/zonemaster/zonemaster-backend/blob/develop/.travis.yml

**Comment by pnax** [comment]
I made some testing on what should the string be to use Perl 5.32 in Travis, see #741.

I tried with :
* `5.32.1` : this fails
* `5.32.0` : this works
* `5.32` : I know this should fail

The list of the Travis jobs :
https://travis-ci.org/github/zonemaster/zonemaster-backend/builds/766931924

**Comment by mattias-p** [comment]
To me this sounds like a usability problem with Travis. Should we really be maintaining our Travis config to keep up with whatever patch version of Perl 5.32 they are providing at the moment?

**Comment by matsduf** [comment]
If we use the function to select Perl version we should at least discover when it goes wrong.

**Comment by mattias-p** [comment]
Yeah I agree, it is a good workaround. I guess I just feel a bit uneasy that we need workarounds like this in the first place.

**Comment by matsduf** [comment]
Resolved by #741
